common/array.h

```diff
@@ -30,6 +30,16 @@
 
 namespace Common {
 
+/**
+ * @defgroup common_array Arrays
+ * @ingroup common
+ *
+ * @brief  Functions for working on arrays.
```

Comment:
these are functions to replace some std arrays

Reply:
I'll correct this (as soon as we agree on terminology).

Reply:
Actually these are classes and not functions.